app/models/public_body.rb

```diff
@@ -714,7 +714,7 @@ def json_for_api
   end
 
   def expire_requests
-    info_requests.find_each(&:expire)
+    info_requests.find_each(&:reindex_request_events)
```

Comment:
-1 to this particular implementation. I'd expect a method called `expire_requests` to call `expire` on the associated requests.

If we need two different reindexing methods in different situations we should create clear and consistent domain language for that.